### PR TITLE
Mitid appswitch login_hint

### PR DESCRIPTION
**Pull request by mickhansen**

### CODE REVIEW

File: src/components/AuthMethodButton/AuthMethodButton.tsx [hunk 1]
```diff
@@ -58,9 +61,16 @@ export default function AuthMethodButton(props: AuthMethodButtonProps) {
   useEffect(() => {
     if (props.href) return;
 
+		let loginHint : string | undefined = undefined;
```
Comment (sgryt) [inline src/components/AuthMethodButton/AuthMethodButton.tsx:64]: Indent-level seems to be off, or is that just GH diff view leading me astray ?

Comment (mickhansen) [reply src/components/AuthMethodButton/AuthMethodButton.tsx:64]: No, that is definitely off, hmm.

**Comment by mickhansen** [comment]
@sgryt Parts of the file were tab intended, i've converted the entire file to space-indented like the rest of the project.

**Comment by mickhansen** [comment]
Works on android with the usual appswitch caveat.

**Pull request merged**